py3status/modules/whatismyip.py

```diff
@@ -13,14 +13,8 @@
         (default 'ip')
     negative_cache_timeout: how often to check again when offline (default 2)
     timeout: how long before deciding we're offline (default 5)
-    url: change IP check url (must output a plain text IP address)
-        (default 'http://ultrabug.fr/py3status/whatismyip')
     url_geo: IP to check for geo location (must output json)
-        (default 'http://ip-api.com/json')
-
-Format placeholders:
-    {country} display the country
-    {ip} display current ip address
```

Comment:
to clarify @tobes, it was not enough to just extend the dict with the `country` as well as `country_name`, because later the exact value is used to format the string - but the adjustment of the `format` itself worked flawlessly.